client/src/components/PetInfo.jsx

```diff
@@ -0,0 +1,73 @@
+//Import necessary functionalities
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { UserStatus } from '../context/UserContext';
+
+const PetInfo = ({ setSelectedPet }) => {
+
+    //Use useNavigate hook from React Router to establish functionality in buttons below
+    const navigate = useNavigate();
+
+    //Import user info for use in data pull
+    const { loggedUser, setLoggedUser } = UserStatus();
+    
+    //State to hold pet information
+    const [petInfo, setPetInfo] = useState(null);
+
+    //Function to load pet information from database
+    const loadPetInfo = async () => {
+        const userId = loggedUser.userId;
+        
+        //Send query with userId as parameter to pull all pets from this user (as primary or secondary user)
+        const response = await fetch(`/api/pets/${userId}`);
+
+        //Save response in a variable to access for display
+        const petData = await response.json();
+
+        if (!petData[0].pets) {
+            setPetInfo(null);
+        } else {
+            //Remove the first element of the array (pet status)
+            petData.shift();
+            //Store the rest of the data (each pet's info) in the petData state
+            setPetInfo(petData);
+        }
+   }
+
+    //Use Effect to download pet info when the component loads
+    useEffect(() => {
+        loadPetInfo();
+    }, []);
+
+    //Function to format pet information in display
+    const formatPetDisplay = () => {
+        return petInfo.map((pet, index) => (
+            <div key={index}>
+                <h4>Pet Name: {pet.name}</h4>
+                <p>Species: {pet.species}</p>
+                <button onClick={() => selectPet(index)}>View Training Plans</button>
+            </div>
+        ))        
+    }
+
+    //Function to set selected pet state for use in UserTrainingPlans component
+    const selectPet = (index) => {
+        //Pull out petId and name
+        const petId = petInfo[index].pet_id;
```

Comment:
Let's talk about this during our next 1-1.  I want to highlight different ways you can do this. I was going to add a comment about creating a getPetInfo method instead of access the object's properties directly, but it's a school of thought with regards to coding standards a team might have.  the other issue is the method formatPetDisplay is tightly coupled to a global object compared to it being passed in. Again there are philosophies around this. 

So, let's talk about it but no need to make changes here.